README.ipynb

```diff
@@ -204,7 +203,7 @@
    },
    "outputs": [],
    "source": [
-    "!pip install -r requirements.txt -c constraints.txt"
+    "!pip install -r requirements.txt"
```

Comment:
i wasnt able to install the packages due to a permission error which was fixed by adding `--user` at the end of this line